pybamm_tea/tea.py

```diff
@@ -303,14 +307,47 @@ def initialize(self):
                     * pava.get("Maximum concentration in positive electrode [mol.m-3]")
                 )
             )
-        self.parameter_values = {**self.parameter_values, **pava}
+        if (
+            pava.get("Initial concentration in negative electrode [mol.m-3]") is None
+            and pava.get("Initial concentration in positive electrode [mol.m-3]") is None
+        ):
+            # initialize concentrations based on initial loss of lithium inventory
+            pava["Initial concentration in negative electrode [mol.m-3]"] = 0
+            pava["Initial concentration in positive electrode [mol.m-3]"] = pava.get("Maximum concentration in positive electrode [mol.m-3]")
+            warnings.warn("Warning: 'Initial concentration in negative electrode [mol.m-3]' and 'Initial concentration in positive electrode [mol.m-3]' are set to 0 and maximum concentration in positive electrode, respectively.")
```

Comment:
In practice the initial concentrations won't be 0 and c_max because you will charge/discharge between voltage limits. I think what you are really doing here is saying that the cyclable lithium is equivalent to starting with a fully lithiated cathode and fully delithiated anode. You could then do something like 
```python3
    param = pybamm.LithiumIonParameters()
    Q_n = parameter_values.evaluate(param.n.Q_init)
    Q_p = parameter_values.evaluate(param.p.Q_init)
    Q_Li = Q_p * f # here 0 < f < 1 gives you initial LLI
    inputs = {"Q_n": Q_n, "Q_p": Q_p, "Q_Li": Q_Li}
    esoh_solver = pybamm.lithium_ion.ElectrodeSOHSolver(parameter_values, param)
    sol = esoh_solver.solve(inputs)

    c_n_max = parameter_values.evaluate(param.n.prim.c_max)
    c_p_max = parameter_values.evaluate(param.p.prim.c_max)
    x = sol["x_100"]
    y = sol["y_100"]
    parameter_values.update(
        {
            "Initial concentration in negative electrode [mol.m-3]": x * c_n_max,
            "Initial concentration in positive electrode [mol.m-3]": y * c_p_max,
        },
        check_already_exists=False,
    )
```

Reply:
The idea was that _if_ no initial concentrations are supplied, the total lithium inventory is what would be supplied with the maximum concentration in the positive electrode before formation, without considering any losses at this stage. Later the initial concentrations are updated to 100% SoC, so that the TEA parameter-set can directly be used for a discharge simmulation. Would you prefer to have the initial lithium inventory set according to 100% SoC (dependent on voltage cut-offs/stoichiometry limits)?